src/FastRoute/src/fastroute.i

```diff
@@ -209,9 +209,9 @@ enable_antenna_avoidance_flow(char* diodeCellName, char* diodePinName)
 }
 
 void
-reset_fastroute()
+clear_fastroute()
```

Comment:
why are you using underscore in the function name, unlike every other function in the file?

Reply:
All functions of this file have underscores

Reply:
right; that is tcl function. sorry